docs/concepts/vscode-extension.md

```diff
@@ -15,7 +15,7 @@ It utilizes the excellent [demisto-sdk](./demisto-sdk) python package.
 
 * Python 3.7 and up.
 * [Python for VSCode](https://marketplace.visualstudio.com/items?itemName=ms-python.python).
-* Install [demisto-sdk](../concepts/demisto-sdk#installation-and-setup). It is highly recommended to install it on a virtual environment such as [pipenv](https://pipenv.pypa.io/en/latest/).
```

Comment:
also add here link to https://xsoar.pan.dev/docs/concepts/dev-setup

Reply:
There's no actual content on this page. Might consider re-work.